src/adapters/scene.js

```diff
@@ -208,13 +208,15 @@ Scene.prototype.initMapBox = async function(locationHash) {
     this.mb.on('dragstart', () => { fire('map_user_interaction'); });
     this.mb.on('pitchstart', () => { fire('map_user_interaction'); });
 
-    this.mb.on('moveend', () => {
-      const { lng, lat } = this.mb.getCenter();
-      const zoom = this.mb.getZoom();
-      store.setLastLocation({ lng, lat, zoom });
-      window.app.updateHash(this.getLocationHash());
-      fire('map_moveend');
-    });
+    if (!window.no_ui) {
+      this.mb.on('moveend', () => {
+        const { lng, lat } = this.mb.getCenter();
+        const zoom = this.mb.getZoom();
+        store.setLastLocation({ lng, lat, zoom });
+        window.app.updateHash(this.getLocationHash());
+        fire('map_moveend');
```

Comment:
This `map_moveend` event is used by the `CategoryPanel` to trigger the search for POIs in the current zone after the intention has been derived from the query and display the markers.  
So it should still be fired. As is no marker would be visible on the map.